Sources/BCRunner/BytecodeRunner.swift

```diff
@@ -72,6 +72,14 @@ private class BytecodeRunner {
   }
 }
 
+public func runBytecode(_ context: CGContext, fromData data: Data) {
```

Comment:
You can use Data.withUnsafeBytes to avoid copying
https://developer.apple.com/documentation/foundation/data/3139154-withunsafebytes